build_tools/azure/install_setup_conda.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ -z "${CONDA}" ]]; then
+    # Determine architecture and OS
+    ARCH=$(uname -m)
+    OS=$(uname -s)
+
+    if [[ "$ARCH" == "arm64" ]]; then
+        MINIFORGE_ARCH="arm64"
+    else
+        MINIFORGE_ARCH="x86_64"
+    fi
+
+    if [[ "$OS" == "Darwin" ]]; then
+        MINIFORGE_OS="MacOSX"
+    else
+        MINIFORGE_OS="Linux"
+    fi
+
+    # Download and install miniforge
+    MINIFORGE_URL="https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-${MINIFORGE_OS}-${MINIFORGE_ARCH}.sh"
+    wget ${MINIFORGE_URL} -O miniforge.sh
+    chmod +x miniforge.sh
+    ./miniforge.sh -b -u -p $HOME/miniforge3
+
+    # Set CONDA environment variable to point to miniforge installation
+    export CONDA=$HOME/miniforge3
+
+fi
+
+# Add conda to PATH
+echo "##vso[task.prependpath]$CONDA/bin"
+
+# Take ownership of conda installation
+sudo chown -R $USER $CONDA
```

Comment:
You need this only when conda is installed so maybe put it in a else clause of `if [[ -z "${CONDA}" ]]`?

IIRC CONDA is a system folder which somehow causes issue somewhere ...